### PR TITLE
Target changed to dev image for devstack for LMS & Studio

### PR DESCRIPTION
**Pull request by zubairshakoorarbisoft**
**Task description:**
https://github.com/openedx/edx-platform/issues/31737

**How to test**
- Go to devstack
- Make sure you have latest edx-platform repo in workspace
- Now, all set to up LMS
- Run `make dev.up.lms`

**Commands to test**
- Make sure `make dev.shell.lms` is running fine and opening shell
- Make sure `make dev.provision.lms` is running fine
- Make sure `make dev.logs.lms` is not throwing any errors, it should run fine and showing logs
- In case, you see issue on [dashboard](http://localhost:18000/dashboard), just open lms shell and run migrations and re-try
- Migration command --> `python manage.py lms migrate`

### CODE REVIEW

**Review by UsamaSadiq — DISMISSED** [review]
We should also look for any other targets using default edxapp_env and update those in the Makefile such as `dev.migrate.lms/studio` etc.

File: docker-compose.yml [hunk 1]
```diff
@@ -429,7 +429,7 @@ services:
       - "44567:4567"
 
   lms:
-    command: bash -c 'source /edx/app/edxapp/edxapp_env && while true; do python /edx/app/edxapp/edx-platform/manage.py lms runserver 0.0.0.0:18000 --settings devstack_docker; sleep 2; done'
```
Comment (iamsobanjaved) [inline docker-compose.yml]: Revert this change for now as we have added compatibility for this in new image

**Review by iamsobanjaved — APPROVED** [review]
